client/src/components/BlogPostForm.tsx

```diff
@@ -64,10 +70,15 @@ export default function BlogPostForm({ entry }: props) {
         photoAuthor: name,
         photoAuthorLink: html,
         photoAlt: alt_description,
-        entryId: undefined,
+        entryId: entryId,
```

Comment:
Some people don't like the following suggestion, but I've gotten used to it and now like it:
Instead of `{ entryId: entryId }`, just use `{ entryId }`. I do this even in a long list of fields like you have here.